strings/slice_test.go

```diff
@@ -0,0 +1,37 @@
+package gxstrings
+
+import (
+	"testing"
```

Comment:
I'd like to know the performance between `Slice(s string) (b []byte)` and `[]byte(s)`, can you provide a bench test for it?

Reply:
ok